Add Kwm 1.1.1

chrishoage opened the pull request:
This adds Kwm to the homebrew binaries. 

Reasons for being a binary and not built from source are as follows: 

It distributes `kwm-overlay`, which depends on the 10.11 SDK. 
It requires accessibility access which must be granted every time the binary is rebuilt from source. 
To avoid this `kwm` has been codesigned, allowing for Accessibility Access to only be granted once. 

Because of these caveats kwm distributes a binary to be used if the source build requirements can't be met.

strages commented:
when will this be merged in? :)

bfontaine commented on kwm.rb:
```diff
@@ -0,0 +1,55 @@
+    bin.install "kwmc"
+    prefix.install "kwm"
+    prefix.install "kwm-overlay"
+    prefix.install "kwmrc"
```
You can merge these three lines with `prefix.install %w[kwm kwm-overlay kwmrc]`.

chrishoage replied:
Addressed this feedback and squashed.

skattyadz commented:
This would be useful for me too

bfontaine commented on kwm.rb:
```diff
@@ -0,0 +1,53 @@
+      mkdir -p ~/.kwm
+      cp #{prefix}/kwmrc ~/.kwm/kwmrc
+    EOS
+  end
```
Sorry for the delay; please remove anything non-Homebrew-specific from here.

chrishoage replied:
Here being `caveats`? My understanding of `caveats` purpose was for post installation instructions. Looking though other formula in homebrew I see similar `cp` commands. 

Can you please clarify what you would like me to remove?

bfontaine replied:
`caveats` are Homebrew-specific instructions. We assume general instructions are already given by upstream.

chrishoage replied:
Searching https://github.com/Homebrew/homebrew/search?utf8=%E2%9C%93&q=caveats+cp

reveals many uses very similar to what I am doing here. Is the issue that it mentions that it requires "accessibility permissions" or that it gives copy commands? Or is the issue both (which if this is the case it is confusing to me due to the precedent set by other formula)

bfontaine replied:
You can probably remove the accessibility issue but keep the other one since it’s specific to the way Homebrew installs it (we don’t install the config in the user’s home directory).

chrishoage replied:
:+1:  thanks

chrishoage commented:
Addressed feedback, however, the package this formula adds has since released version `1.1.1`. 

What is the best course of action for me to take: Update this PR to the latest version, or submit an updated PR with the new version after this one lands? 

Thank you for your time!

dunn commented:
Go ahead and bump the version and force-push to your branch to update this PR.  Thanks!

chrishoage commented:
I have bumped the version, and updated the commit message.

dunn commented:
Thanks! Merged in a00d85df51acd1e9bcfbb0093b876a0a1139050c.
